app/models/knight.rb

```diff
@@ -1,15 +1,16 @@
 class Knight < Piece
-  def is_obstructed?(dest_x, dest_y)
+  def is_obstructed?(_dest_x, _dest_y)
     false
   end
 
   def valid_move?(dest_x, dest_y)
     delta_x = (dest_x - x_position).abs
     delta_y = (dest_y - y_position).abs
-    dest_piece = game.pieces.where("x_position = ? AND y_position = ?", dest_x, dest_y).take
-    # byebug
+    dest_piece = game.pieces
+                 .where("x_position = ? AND y_position = ?", dest_x, dest_y)
+                 .take
```

Comment:
I've really had ENOUGH of this Jeff. You take a line of code that is perfectly VALID and READABLE by HUMANS and split it to appease a piece of SOFTWARE that is counting width of a column.  Please desist with this "well I'll just fix that for you" mentality and stop editing my code.  I have held my breath and bitten my tongue one time too many, this is exceptionally bad form on your part.